bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -66,6 +66,9 @@ struct ReplicaConfig {
   // a time interval in milliseconds, represents the timeout for automatically replacing the primary
   uint16_t autoPrimaryRotationTimerMillisec = 0;
 
+  // preExecutionFeatureEnabled=0/1; preExecutionFeatureEnabled=1 enables the pre-execution feature
+  uint8_t preExecutionFeatureEnabled = 0;
```

Comment:
why not `bool` ?

Reply:
Do we like to serialize booleans?

Reply:
Iknow that we do that, but it is a bad practice.

Reply:
I'd prefer to be consistent with the rest of the code - and do an eventual clean-up separately.

Reply:
Hm. OK.